srcalc/src/main/java/gov/va/med/srcalc/domain/VistaPerson.java

```diff
@@ -1,42 +1,45 @@
 package gov.va.med.srcalc.domain;
 
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Optional;
+
 /**
  * A VistA person (from the NEW PERSON file).
  */
 public class VistaPerson
 {
-    private final String fDivision;
+    private final String fStationNumber;
     private final String fDuz;
     private final String fDisplayName;
-    private final String fUserClass;
+    private final Optional<String> fProviderType;
```

Comment:
Yes, Person Classes are clinical categories. Non-clinical users in VistA may have a "User Class" which is a different attribute.